### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by axelstudios**
#### What's this PR do?
- Updates linux documentation
- Updates js dependencies
- Updates and simplifies ui-grid decorators
- Added ColumnListSettingFilter
- Fixed landing background for ultrawide monitors
- Removed deprecated saving indicator from header
- Consistent trademark usage
- Consistent landing page content
- Fixed long-standing bug where mapping suggestion typeahead would match too many columns.  For instance typing "state" would match every column.
- Prevented autocomplete from overriding typeahead
- Improved UI handling of new org creation
- Replaced custom NoPagination class with better alternative
- Remove notes pagination and fixed notes horizontal scrolling
- Fixed `create_default_user` bug where capital letters in email would prevent login
- Lint fixes
- Added chunked inventory loading
#### How should this be manually tested?
1. `npm install`
2. Monkey test, and check inventory loading for organizations with 5,000+ records
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: seed/data_importer/tasks.py [hunk 1]
```diff
@@ -219,10 +219,14 @@ def _translate_unit_to_type(unit):
 
     query_set = Column.objects.filter(organization=org, units_pint__isnull=False)
     for column in query_set:
+        # DON'T OVERRIDE DEFAULT COLUMNS WITH DATA FROM RAW COLUMNS
```
Comment (nllong) [inline seed/data_importer/tasks.py]: ha, nice

File: seed/management/commands/create_default_user.py [hunk 2]
```diff
@@ -52,13 +52,13 @@ def handle(self, *args, **options):
 
             if options['usertype'] == 'superuser':
                 u = User.objects.create_superuser(
-                    options['username'],
+                    options['username'].lower(),
```
Comment (nllong) [inline seed/management/commands/create_default_user.py]: oh, very nice if using the command line to create the user. We probably still have users with both lower case and mixed case names in the database!

File: seed/models/column_list_settings.py [hunk 3]
```diff
@@ -15,25 +15,25 @@
     Column,
 )
 
+VIEW_LIST = 0
```
Comment (nllong) [inline seed/models/column_list_settings.py]: Any reason to not scope this?

File: seed/models/data_quality.py [hunk 4]
```diff
@@ -79,7 +79,7 @@ def format_pint_violation(rule, source_value):
         formatted_min = '{:.1f} {}'.format(rule.min, pretty_rule_units)
     if rule.max is not None:
         formatted_max = '{:.1f} {}'.format(rule.max, pretty_rule_units)
-    return (formatted_value, formatted_min, formatted_max)
+    return formatted_value, formatted_min, formatted_max
```
Comment (nllong) [inline seed/models/data_quality.py]: thanks python.

File: seed/serializers/column_list_settings.py [hunk 5]
```diff
@@ -30,9 +32,9 @@ class Meta:
 
 
 class ColumnListSettingSerializer(serializers.ModelSerializer):
-    columns = ColumnListSettingColumnSerializer(source="columnlistsettingcolumn_set", read_only=True, many=True)
-    settings_location = ChoiceField(choices=ColumnListSetting.VIEW_LOCATION_TYPES)
-    inventory_type = ChoiceField(choices=ColumnListSetting.VIEW_LIST_INVENTORY_TYPE)
+    columns = ColumnListSettingColumnSerializer(source='columnlistsettingcolumn_set', read_only=True, many=True)
+    settings_location = ChoiceField(choices=VIEW_LOCATION_TYPES)
```
Comment (nllong) [inline seed/serializers/column_list_settings.py]: disregard my scoping comment.

File: seed/tests/test_note_views.py [hunk 6]
```diff
@@ -56,8 +56,7 @@ def test_get_notes_property(self):
         url = reverse('api:v2.1:property-notes-list', args=[self.pv.pk])
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        result = json.loads(response.content)
-        results = result['results']
```
Comment (nllong) [inline seed/tests/test_note_views.py]: ha, nice catch.

File: seed/views/labels.py [hunk 7]
```diff
@@ -59,7 +58,7 @@ class LabelViewSet(DecoratorMixin(drf_api_endpoint), viewsets.ModelViewSet):
     parser_classes = (JSONParser, FormParser)
     queryset = Label.objects.none()
     filter_backends = (LabelFilterBackend,)
-    pagination_class = NoPagination
+    pagination_class = None
```
Comment (nllong) [inline seed/views/labels.py]: much cleaner.

**Comment by nllong** [comment]
Also can you add the issues that this is related too? Is #1601 one of these?

File: seed/management/commands/create_default_user.py (hunk 2 above)
Comment (axelstudios) [reply seed/management/commands/create_default_user.py]: Yeah, more than a few times I used capital letters in my email address with that command and was prevented from logging in until I updated the database value 🤦‍♂️

**Comment by axelstudios** [comment]
@nllong Just added chunked loading to the `Show Populated Columns` dialog and the inventory map view, so that's ready to test

**Comment by nllong** [comment]
Testing now. Do you want me to merge if all is well?

**Comment by nllong** [comment]
Tested. Looks good!

**Pull request merged**